collector/os.go

```diff
@@ -261,12 +257,21 @@ func (c *OSCollector) collect(ctx *ScrapeContext, ch chan<- prometheus.Metric) (
 		timezoneName,
 	)
 
-	ch <- prometheus.MustNewConstMetric(
-		c.PagingFreeBytes,
-		prometheus.GaugeValue,
-		pfb,
-	)
-
+	if pagingErr == nil {
+		ch <- prometheus.MustNewConstMetric(
+			c.PagingFreeBytes,
+			prometheus.GaugeValue,
+			pfb,
+		)
+
+		ch <- prometheus.MustNewConstMetric(
+			c.PagingLimitBytes,
+			prometheus.GaugeValue,
+			fsipf,
+		)
+	} else {
+		log.Debugln("Could not find SYSTEM\\CurrentControlSet\\Control\\Session Manager\\Memory Management key. windows_os_paging_free_bytes and windows_os_paging_limit_bytes will be omitted.")
```

Comment:
Could the fact that this is in the hklm hive please be reflected into the log message? There are several to choose from when verifying the system state vs code behaviour.

```suggestion
		log.Debugln("Could not find HKLM:\SYSTEM\\CurrentControlSet\\Control\\Session Manager\\Memory Management key. windows_os_paging_free_bytes and windows_os_paging_limit_bytes will be omitted.")
```

Or extract a variable for the paths as well?

Reply:
That's a good idea, I'll add `HKLM` to the log message.